.github/ISSUE_TEMPLATE/azure_snapshot_debugger_issue.md

```diff
@@ -1,6 +1,14 @@
 ---
 name: Issue with Application Insights Snapshot Debugger
 about: Need help with Application Insights Snapshot Debugger
+labels: snapshot-collector
 ---
 
-If you're seeing issue with [Application Insights Snapshot Debugger](https://docs.microsoft.com/azure/azure-monitor/app/snapshot-debugger), get support from [Azure Support Center](https://azure.microsoft.com/support/create-ticket/).
+If you encounter an issue with [Application Insights Snapshot Debugger](https://docs.microsoft.com/azure/azure-monitor/app/snapshot-debugger) in Azure, please use the [Azure Support Center](https://azure.microsoft.com/support/create-ticket/) to file a ticket. Also, check the [troubleshooting documentation](https://docs.microsoft.com/azure/azure-monitor/app/snapshot-debugger-troubleshoot). Otherwise, use this template to report a bug or ask questions about the Microsoft.ApplicationInsights.SnapshotCollector NuGet.
```

Comment:
I would vote for removing "Otherwise, use this template to report a bug or ask questions about the Microsoft.ApplicationInsights.SnapshotCollector NuGet" since the nuget package is not coming from this repo.

Reply:
Consider this:

```markdown
If you encounter an issue with [Application Insights Snapshot Debugger](https://docs.microsoft.com/azure/azure-monitor/app/snapshot-debugger) in Azure, please check the [troubleshooting documentation](https://docs.microsoft.com/azure/azure-monitor/app/snapshot-debugger-troubleshoot). Use the [Azure Support Center](https://azure.microsoft.com/support/create-ticket/) to file a ticket if the troubleshooting document didn't help. We will close the issue related to Snapshot Debugger since this repository is only covering ApplicationInsights .NET SDK.
```